models/commit_status.go

```diff
@@ -147,18 +147,22 @@ func GetCommitStatuses(repo *Repository, sha string, opts *CommitStatusOptions)
 	if opts.Page <= 0 {
 		opts.Page = 1
 	}
+	if opts.PageSize <= 0 {
```

Comment:
How about add a method of `ListOptions` named `SetDefault` includes line 147 to 152?

Reply:
I think this is the only scenario where `opts.PageSize = ItemsPerPage`. There are other cases which `opts.PageSize` is hardcoded but it is not normalized... like here:  
https://github.com/go-gitea/gitea/blob/7a3a90aca3c4a8258805efa10022a81e4d1a4d73/routers/api/v1/repo/issue.go#L74